### ISSUE TITLE
Autocomplete no longer working on PHPStorm 2018.3

### ISSUE THREAD

**Issue by rrajkomar** [Haehnchen/idea-php-symfony2-plugin#1266]
Hi, 
Since the latest release of PHPStorm it seems the autocomplete for repository methods, entity manager methods seems to no longer be working.
Here are my configuration information : 

PhpStorm 2018.3
Build #PS-183.4284.150, built on November 21, 2018
JRE: 1.8.0_152-release-1343-b15 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.9.0-8-amd64

Plugin version : v0.17.169

Symfony 3.4

**Comment by alturic** [comment]
Are you referring to Twig template? If so, yes I'm having the same issue. If not, all of my auto-completion is working except for Twig entity properties...

**Comment by bcastagna** [comment]
Uninstalling and re-installing the plugin from the market worked for me. Go figure...

**Comment by rrajkomar** [comment]
@alturic : no I meant autocompletion for methods like `em->getRepository(...)->getSomeMethod()` 
Also from a controller a $this->get(...) does not seem to be recognized

@bcastagna : try uninstalling/reinstalling did not change a thing :-(

**Comment by rrajkomar** [comment]
I saw a new version version of the plugin was released but the issue seems to still be there

**Comment by rrajkomar** [comment]
Ok foudn why it wasn't working for me : when you have multiple vendors directories in the same project and the same classes are present in several of those, PHPStorm can't properly provide the necessary data for the autocompelte to work.

I had to exclude all but one of my vendors directories (didn't like it though) in order for it to work again.

**Comment by er1z** [comment]
What is the point of having more than single `vendor`? Just curious. :D

**Comment by rrajkomar** [comment]
Hi, not my choice.
But the answer is monolithic repository with several closely related (yet distinct) application projects (each with a specific vendor dir)

**Comment by er1z** [comment]
If each sub-project is in another directory, just open it within anothe PHPStorm project.

**Comment by Jeordy** [comment]
Same problem here. EntityManager autocomplete does work in Symfony 4 projects, but not in 3.4 anymore. Reinstalled the plugin, did a reindex, removed .idea folder and recreated the project, updated composer, etc. Nothing seems to work. Any help is really appreciated.

**Comment by bogdandovgopol** [comment]
Same here.

**Comment by mortolian** [comment]
I started a new Symfony 5 project with PhpStorm 2019.3 and Symfony Support Plugin 0.19.189. Autocomplete did not work properly and namespaces did not resolve.

What fixed the problem for me was to right-click on the project name and then select 'Inspect Code'.
![Screenshot 2020-03-08 at 10 27 51](https://user-images.githubusercontent.com/12611057/76159370-6d62d080-6128-11ea-9c5a-d0e5ad5db85e.png)

And voilà!

![Screenshot 2020-03-08 at 10 29 04](https://user-images.githubusercontent.com/12611057/76159399-c7639600-6128-11ea-94c1-5de08254abfb.png)

What is also worth mentioning is that the problem started after an upgrade of PhpStorm, so that is something worth pondering.

Things I tried that did not work:

* Removing the Symfony Support and associated plugins and re-installing them, 
* Restarting in all kinds of different ways, 
* Removing the project and recreating it from scratch,
* Composer dump-autoload
